client/src/components/Dashboard/CreateCapsule.jsx

```diff
@@ -1,18 +1,21 @@
 import { AddIcon } from '@chakra-ui/icons'
 import { Box, Flex, Text } from '@chakra-ui/react'
 import { useRouter } from 'next/router'
+import { useMediaQuerySSR } from 'hooks'
 import React from 'react'
 import EditButton from 'components/Dashboard/League/EditButton'
 
 const CreateCapsule = ({ heading, borderRadius, buttonNum }) => {
     const router = useRouter()
+    const isDesktop = useMediaQuerySSR(940)
     const buttonNums = { 0: '12fr', 1: '12fr 1fr', 2: '12fr 1fr 1fr' }
+
     return (
         <Box display="grid" gridTemplateColumns={buttonNums[buttonNum]} w="100%">
             <Flex
                 w="100%"
                 h="70px"
-                borderRadius={borderRadius}
+                borderRadius={isDesktop ? borderRadius : '24px'}
```

Comment:
not a big deal but chakra ui can handle this kind of case, instead u can do
`borderRadius={['24px', borderRadius]}` for the same-ish effect

Reply:
happy to explain how it works if u want to know more